snaps/features/custom-evm-accounts/index.md

```diff
@@ -273,17 +274,89 @@ It resolves the request using
 and the Snap resolves the request using [`snap_manageAccounts`](../../reference/snaps-api.md#snap_manageaccounts),
 notifying MetaMask of the result.
 
-## Supported signing methods
+## EOA methods
 
-An account management Snap can implement support for handling the [`personal_sign`](/wallet/reference/personal_sign),
-[`eth_signTypedData_v4`](/wallet/reference/eth_signtypeddata_v4), and
-[`eth_signTransaction`](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_signtransaction)
-Ethereum signing methods.
+An account management Snap can implement the following methods to support dapp requests from
+externally owned accounts (EOAs):
 
-The Snap can also implement support for [deprecated signing
-methods](/wallet/concepts/signing-methods/#deprecated-signing-methods) that some dapps still
-might use.
+- [`personal_sign`](/wallet/reference/personal_sign)
+- [`eth_signTypedData_v4`](/wallet/reference/eth_signtypeddata_v4)
+- [`eth_signTransaction`](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_signtransaction)
+- [Deprecated signing methods](/wallet/concepts/signing-methods/#deprecated-signing-methods)
 
-## Example
+## Account abstraction (ERC-4337)
 
-See the [example account management Snap source code](https://github.com/MetaMask/snap-simple-keyring) for more information.
+:::flaskOnly
+:::
+
+Account abstraction, specified by [EIP-4337](https://eips.ethereum.org/EIPS/eip-4337), introduces
+*user operations* and enables users to manage smart contract accounts containing arbitrary
+verification logic.
+Users can use these ERC-4337 accounts instead of externally owned accounts as primary accounts.
```

Comment:
Maybe introduce the official link for ERC on first mention? I understand the ERC is a subset of the EIP, but might be helpful to have both sources presented.

```suggestion
Users can use these [ERC-4337](https://www.erc4337.io) accounts instead of externally owned accounts as primary accounts.
```